motoman_gp200r_support/urdf/gp200r_macro.xacro

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="motoman_gp200r" params="prefix">
+
+    <!-- Properties -->
+    <material name="yaskawa_blue">
+        <color rgba="0 0.35546875 0.671875 1" />
+    </material>
+
+    <!-- link list -->
+    <link name="${prefix}base_link">
+        <visual>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/base_link.STL" scale="0.001 0.001 0.001"/>
+            </geometry>
+            <material name="yaskawa_blue"/>
+        </visual>
+        <collision>
+            <geometry>
+                <mesh filename="package://motoman_gp200r_support/meshes/gp200r/visual/base_link.STL" scale="0.001 0.001 0.001"/>
```

Comment:
This appears to use `visual` quality meshes for `collision` purposes. That's not recommended. Performance will suffer, especially in motion planners like MoveIt.

Please create suitable collision meshes. You could create convex hulls using Blender or some other mesh editing tool.

Reply:
I have exported convex hulled .stl's from Blender to use for collision in the urdf.